server/controllers/stripe.js

```diff
@@ -8,6 +11,50 @@ exports.createUser = asyncHandler(async (req, res, next) => {
   const user = await User.findById(req.user.id);
   const profile = await Profile.findOne(req.user);
 
+  const schedule = await Schedule.create({
+    name: 'default schedule',
+    periods: {
+      monday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      tuesday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      wednesday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      thursday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      friday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      saturday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      sunday: {
+        startTime: '10',
+        endTime: '22',
+      },
+    }
+  });
+
+  const availability = await Availability.create({
+    scheduleIds: [schedule._id],
+    activeScheduleId: schedule._id,
+  });
```

Comment:
I get this is for testing - could we also remove this for now 🤔

Reply:
removed in latest version